.circleci/config.yml

```diff
@@ -170,13 +170,11 @@ jobs:
             echo "$failures"
 
             echo "Sending metrics:"
-            # Passed [sfdx/stable-rc/installer/macos - https://circleci.com/gh/salesforcecli/status/925] -> count#platform-cli.sfdx.stable-rc.installer.macos.success=1
```

Comment:
Can you keep these comments in here? Otherwise, it's difficult to understand all the processing that's happening here:

```
base="platform-cli".$(echo $line | grep -oP '(?<=\[).*(?=\])' | cut -f1 -d"|" | tr '/' '.' | xargs)
```